forms-flow-web/src/apiManager/services/processServices.js

```diff
@@ -66,10 +66,12 @@ export const getProcessStatusList = (processId, taskId) => {
 export const fetchAllBpmProcesses = (...rest) => {
   const done = rest.length ? rest[0] : () => {};
   return (dispatch) => {
-    httpGETRequest(API.PROCESSES, {}, UserService.getToken(), true)
+    httpGETRequest(API.GET_BPM_PROCESS_LIST + "?latestVersion=true&excludeInternal=true", {}, UserService.getToken(), true)
```

Comment:
GET_BPM_PROCESS_LIST has a v1 in it and we are requesting latest version again

Reply:
It is just used to indicate the REST API change versioning. latestVersion is only passed to camunda to get latest data. In the future if there are some drastic changes happens then we release v2, v1 remains the same. This will avoid the code breakage